controller/src/main/java/org/jboss/as/controller/audit/InMemoryAuditLogHander.java

```diff
@@ -58,36 +58,34 @@
     }
     private static final String IN_MEMORY_FORMATTER_NAME = "in-memory-formatter";
     private final List<ModelNode> items;
-    private int maxHistory;
+    private volatile int maxHistory;
```

Comment:
The rest of this class isn't thread safe. I don't think `volatile` is needed here.